reactivesocket-core/src/test/java/io/reactivesocket/frame/LeaseFrameFlyweightTest.java

```diff
@@ -0,0 +1,19 @@
+package io.reactivesocket.frame;
+
+import org.agrona.concurrent.UnsafeBuffer;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+import static org.junit.Assert.*;
+
+public class LeaseFrameFlyweightTest {
```

Comment:
There is a test in FrameHeaderFlyweightTest that test encoding a frame at MAX_SIZE and then that it fails at MAX_SIZE + 1. I think that's the best you're going to get :)